deploy/docker/edumfa.py

```diff
@@ -1,7 +1,21 @@
 from os import getenv
 
 
-def _getenv(key: str, default: str | None = None) -> str:
+def get_content_from_file(path: str) -> str:
+    """Return the content from a path. Strip newlines.
+
+    :param path: The path to read the file from.
+    :return: The content of the file stripped of newlines.
+    :raises FileNotFoundError: If the file at path can't be found.
+    """
+    with open(path) as fp:
+        content = fp.read()
+    # remove any leading/trailing newlines
+    content = content.strip("\n")
+    return content
```

Comment:
```suggestion
    """Return the normalized content from a path.

    :param path: The path to read the file from.
    :return: The content of the file stripped of newlines.
    :raises FileNotFoundError: If the file at path can't be found.
    """
    with open(path) as fp:
        return fp.read().strip()
```
`.strip()` without an arg is better here, since it cleans all whitespace chars